xar/xar_util.py

```diff
@@ -19,15 +19,18 @@
 import tempfile
 import time
 import uuid
+import grp
 
 
 logger = logging.getLogger("xar")
 
-if "centos" in platform.platform().lower():
-    NOGROUP = "nobody"
-else:
-    # Works for debian and darwin for sure
-    NOGROUP = "nogroup"
+try:
+    NOGROUP = grp.getgrnam("nogroup").gr_name
+except KeyError:
+    try:
+        NOGROUP = grp.getgrnam("nobody").gr_name
+    except KeyError:
+        raise
```

Comment:
There is no point to catch and raise - Might as well let it just raise :). Example:

```python
try:
    NOGROUP = grp.getgrnam("nogroup").gr_name
except KeyError:
    NOGROUP = grp.getgrnam("nobody").gr_name
```

Reply:
@cooperlees good point! fixed it